poc_iot_verifier/Cargo.toml

```diff
@@ -51,3 +51,4 @@ rust_decimal = {workspace = true, features = ["maths"]}
 rust_decimal_macros = {workspace = true}
 humantime = {workspace = true}
 twox-hash = {workspace = true}
+itertools = "*"
```

Comment:
We are also using `itertools` in `density_scalar` so we should probably move `itertools` up to the main `Cargo.toml`.

Reply:
fixed here: db51b68a7b91454cc309fa0e92ea7e75463c3e01